app/routes/admin/OverviewRoute.tsx

```diff
@@ -6,19 +6,15 @@ import replaceUnlessLoggedIn from 'app/utils/replaceUnlessLoggedIn';
 import email from './email';
 import groups from './groups';
 
-const OverviewRoute = ({
-  children,
-  currentUser,
-  loggedIn,
-  match,
-}: {
-  children: any;
+interface Props {
```

Comment:
I feel like we should be consist of our use of `interface`/`type`.

Reply:
Do you have a specific preference? I usually use interface whenever I can, but I rarely use any interface-specific features.

Reply:
I feel like we usually use `type` on props here, so I guess I prefer that in terms of consistency.